guidelines/code-review.md

```diff
@@ -0,0 +1,15 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# Code Review Guidelines
+
+Short summary of the code review process in the Beman Project organization.
+
+* As stated in the Beman Standard, [REPOSITORY.CODEOWNERS](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_STANDARD.md#repositorycodeowners) is a REQUIREMENT, thus a list of local reviewers is available for each repository.
+   - Note: If you find a repository without a CODEOWNERS file or without a rule set (e.g., PR and code review enforced before merging), please report to Beman Leads or on [Discourse](https://discourse.bemanproject.org).
+
+* External contributors require a review from a code owner. If the tagged owner is not available, please ping another one or request a review on [Discourse](https://discourse.bemanproject.org).
+
+* Internal contributors also require a review from a code owner. As an exception, if no review is received within 24 hours, the code owner can merge the PR, but notify the reviewers. Any reported issues will be fixed by the same code owner.
+  - Note: Internal contributors require to have admin rights to bypass the review process in exception cases. Please ping Beman Leads or on [Discourse](https://discourse.bemanproject.org) to request admin rights ahead.
```

Comment:
I mean if they are a repo owner I think they have these rights? I'm not sure this note adds anything?

Reply:
That's not true. It depends on 1) who created the repo (e.g., it was created by X on personal account and then moved or it was created by an org admin and 2) what are the rules set in that repo (by default, admin cannot bypass rules, but they can be configured so):

![image](https://github.com/user-attachments/assets/4cf84595-f7e9-4732-8452-bed01c5c92b2)

Reply:
that's fair -- some suggested wordsmitting...